stac_fastapi/extensions/stac_fastapi/extensions/core/fields/request.py

```diff
@@ -70,16 +70,11 @@ def filter_fields(self) -> Dict:
         }
 
 
-@dataclass
+@attr.s
 class FieldsExtensionGetRequest(APIRequest):
     """Additional fields for the GET request."""
 
-    fields: Annotated[Optional[str], Query()] = None
-
-    def __post_init__(self):
-        """convert attributes."""
-        if self.fields:
```

Comment:
🤦 I think the whole issue came from this.

I should have added `super().__post_init__()` here (maybe) 

the attrs solution is definitely better anyway